core-bundle/src/EventListener/DataContainer/ContentCompositionListener.php

```diff
@@ -104,7 +104,7 @@ public function renderPageArticlesOperation(array $row, ?string $href, string $l
         $pageModel->setRow($row);
 
         if (!$this->pageRegistry->supportsContentComposition($pageModel) || !$this->hasArticlesInLayout($pageModel)) {
-            return $this->image->getHtml(preg_replace('/\.svg$/i', '_.svg', $icon)).' ';
+            return null !== $icon ? $this->image->getHtml(preg_replace('/\.svg$/i', '_.svg', $icon)).' ' : '';
```

Comment:
This check is not necessary, because if `$icon` is `null`, we are returning early in line 98, aren't we?

Reply:
Line 98 only returns if both `$href` _and_ `$icon` are `null`. Granted, usually none or both will be null, but we should employ defensive programming anyway, should we not?

Reply:
You are right.